.github/workflows/ci.yml

```
@@ -21,16 +21,17 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)